### PR TITLE
remove `where`

### PR DESCRIPTION
**Pull request by felixhaeberle**
hi folks,

this is a PR which removes `where`because I think this is not needed anymore.

do you agree? :+1: / :-1:

### CODE REVIEW

File: source-code/core2/src/fullExample.ts [hunk 1]
```diff
@@ -96,11 +96,11 @@ message.body["en-US"] = {
 }
 
 inlang.messages.query.update({
-	where: { id: "myMessageId" },
+	id: "myMessageId",
 	data: message,
```
Comment (ivanhofer) [inline source-code/core2/src/fullExample.ts:100]: why do we need a separate `data` prop?

**Review by ivanhofer — APPROVED** [review]

File: source-code/core2/src/fullExample.ts (hunk 1 above)
Comment (samuelstroschein) [reply source-code/core2/src/fullExample.ts:100]: mixing `id` with `data` would likely become confusing, don't you think?

**Review by samuelstroschein — APPROVED** [review]

File: source-code/core2/src/fullExample.ts (hunk 1 above)
Comment (ivanhofer) [reply source-code/core2/src/fullExample.ts:100]:
```ts
inlang.messages.query.update({
	id: "myMessageId",
	body: [/* */],
})
```
?

**Review by samuelstroschein — COMMENTED** [review]
The `where` clause exists to update the query API over time. 

```ts
const messages = query({ where: { body: { pattern: {  includes: "hello-world" })
```

Hence, accepting this proposal makes current query requirements simpler, potential future requirements difficult. I go with the majority vote.

**Comment by felixhaeberle** [comment]
This could be a simple filter in the app? Most apps will need to filter in some sort anyway.

**Comment by NilsJacobsen** [comment]
I feel like removing these filter options just makes it harder in the future. The tradeoff is small.

```ts
const messages = query({ where: { body: { pattern: {  includes: "hello-world" })
```

**Comment by felixhaeberle** [comment]
I'll close this. Leaving it in gives the possibility for further advanced use cases.

**Pull request closed without merging**